### PR TITLE
Fixes drone light toggle while dead

### LINKED ISSUE #30411: Drones can toggle light when they're dead.
Self-explanatory title.

### PR DESCRIPTION
**Pull request by kingofkosmos**
Drones are not able to toggle light anymore while dead.

Fixes #30411

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/verbs.dm [hunk 1]
```diff
@@ -15,14 +15,15 @@
 /mob/living/simple_animal/drone/verb/toggle_light()
 	set category = "Drone"
 	set name = "Toggle drone light"
-	if(light_on)
-		set_light(0)
-	else
-		set_light(8)
+	if(stat != DEAD)
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]:
```
if(stat == DEAD)
    to_chat(src, "<span class='userdanger'>There's no light in your life... by that I mean you're dead.</span>")
```

**Comment by Bawhoppen** [comment]
Actually it was kind of interesting for a broken malfunction husk of a drone to be able to have its light flicker

**Comment by RemieRichards** [comment]
@Bawhoppen 
"drone is Urist McDumb a traitor?, flash once for yes, two for no"
*light on*
*light off*

**Comment by Bawhoppen** [comment]
Yes I know the argument, but I was just saying it was cool.

Though to be fair, can't we just ban the people who cheat?

**Comment by RemieRichards** [comment]
we can but then we'd need to bring back ghost chairs

**Comment by RemieRichards** [comment]
also if you just want the coooool~ of it we can make the drone automatically do it or something.
Though I'd want a new type of light before we do anything like that (one optimised for flickering, if possible)

**Comment by lzimann** [comment]
it should be either automatic or not happen, players shouldn't have control on any dead mob tbh

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**